OpenRA.Game/GameRules/MusicInfo.cs

```diff
@@ -42,9 +42,9 @@ public void Load()
 			using (var s = Game.ModData.ModFiles.Open(Filename))
 			{
 				if (Filename.ToLowerInvariant().EndsWith("wav"))
```

Comment:
Wait. I was told to close my PR because this one supersedes it, but now that I look at this, it's missing ... everything from my PR. Namely removing this hardcoded 2-file-types assumption.